package.json

```
@@ -22,21 +22,19 @@
   "bugs": {
     "url": "https://github.com/Brightspace/frau-xsrf-token/issues"
   },
-  "dependencies": {
-    "superagent": "^7"
-  },
   "devDependencies": {
-    "chai": "^3.4.0",
-    "chai-as-promised": "^5.1.0",
-    "coveralls": "^2.11.4",
-    "eslint": "^1.7.3",
-    "eslint-config-brightspace": "0.0.3",
-    "eslint-plugin-react": "^3.5.0",
-    "istanbul": "^0.4.0",
-    "mocha": "^2.3.3",
-    "nock": "^2.15.0",
-    "sinon": "^1.17.2",
-    "sinon-chai": "^2.8.0",
-    "superagent-mock": "^1.7.0"
-  }
+    "@open-wc/testing": "^3",
+    "@web/test-runner": "^0.13",
+    "eslint": "^8",
+    "eslint-config-brightspace": "^0.17",
```

```suggestion
    "eslint-config-brightspace": "^0.18",
```
Or does this break a lot and you're leaving this for a separate upgrade?

---

Done. Haha wasn't intentional -- I just started these changes so long ago that the version changed in between.